misk/src/test/kotlin/misk/tracing/ClientServerTraceTest.kt

```diff
@@ -68,7 +67,7 @@ internal class ClientServerTraceTest {
     val client = clientInjector.getInstance<ReturnADinosaur>(Names.named("dinosaur"))
     client.getDinosaur(dinosaurRequest).execute()
 
-    val serverSpan = serverTracer.take()
+    val serverSpan = serverTracer.take("http.action")
```

Comment:
I needed to add these parameters because I introduced a ScheduledExecutorService above, and that is automatically traced, and those traces fight with this test.